### ISSUE TITLE
about the index of continous features

### ISSUE THREAD

**Issue by ssdyue** [chenxijun1029/DeepFM_with_PyTorch#7]
in data/dataset.py

```
if self.train:
            # index of continous features are zero
            Xi_coutinous = np.zeros_like(dataI[:continous_features])
else:
            # index of continous features are one
            Xi_coutinous = np.ones_like(dataI[:continous_features])
```

Why are the indexes generated by the continuous variables of the training set and the test set zero and one respectively? And should index for each continuous variable be the same?
